app/controllers/images_controller.rb

```diff
@@ -1,13 +1,15 @@
+# frozen_string_literal: true
+
 class ImagesController < ActionController::Base
   PUBLIC_MOUNTS = {
-    'User' => :image,
-  }
+    'User' => :image
+  }.freeze
```

Comment:
this is unnecessary clutter, no freeze please

Reply:
I can remove the rule 👍

Reply:
I would add freeze on CONSTANTS. I know, the benefits are marginal in this app, but I got so used to it already...